Geometry/MTDGeometryBuilder/src/MTDGeomBuilderFromGeometricTimingDet.cc

```diff
@@ -103,14 +103,23 @@ void MTDGeomBuilderFromGeometricTimingDet::buildPixel(
 
   // this is a hack while we put things into the DDD
   int ROCrows(0), ROCcols(0), ROCSx(0), ROCSy(0);
+  int GAPxInterpad(0), GAPxBorder(0), GAPyInterpad(0), GAPyBorder(0);
   switch (det) {
     case GeomDetType::SubDetector::TimingBarrel:
+      GAPxInterpad = ptp.vitems_[0].vpars_[0];
+      GAPxBorder = ptp.vitems_[0].vpars_[1];
+      GAPyInterpad = ptp.vitems_[0].vpars_[2];
+      GAPyBorder = ptp.vitems_[0].vpars_[3];
```

Comment:
Please add a comment here documenting that the border and interpad values are in microns (or whatever the correct unit  is).